### PR TITLE
[Workers] Remove Cache API per-request storage limit

### PR DESCRIPTION
**Pull request by fhanau**
This has been removed in https://github.com/cloudflare/workerd/pull/1863, which has been deployed to production since.
Also updates the table formatting to list the unbound and standard models together – currently the Standard column can't be seen without scrolling to the right which is confusing (https://developers.cloudflare.com/workers/platform/limits/#cache-api-limits), putting them together also matches the approach used for the table above.

### CODE REVIEW

File: content/workers/platform/limits.md [hunk 1]
```diff
@@ -92,18 +92,15 @@ On the Unbound billing model, scheduled Workers ([Cron Triggers](/workers/config
...
+| Feature                       | Workers Free  | [Bundled](/workers/platform/pricing/#example-pricing-bundled-usage-model) | [Unbound](/workers/platform/pricing/#example-pricing-unbound-usage-model) and [Standard](/workers/platform/pricing/#example-pricing-standard-usage-model) |
+| ----------------------------- | ------------- | ------- | ------- |
+| [Max object size](#cache-api-limits) | 512 MB | 512 MB  | 512 MB  |
+| [Calls/request](#cache-api-limits)   | 50     | 50      | 1,000   |
 
 {{</table-wrap>}}
 
 - 50 total `put()`, `match()`, or `delete()` calls per-request, using the same quota as `fetch()`.
 
-- 5 GB total `put()` per request.
```
Comment (fhanau) [inline content/workers/platform/limits.md:105]: @harrishancock Should we note that there used to be a per-request limit here?

**Comment by irvinebroque** [comment]
Think these limits are still relevant to show on this page, since there is still a default limit, and the only change here is the mechanics of where and how the limit is defined (internal implementation detail) — is that accurate?

**Comment by fhanau** [comment]
> Think these limits are still relevant to show on this page, since there is still a default limit, and the only change here is the mechanics of where and how the limit is defined (internal implementation detail) — is that accurate?

https://github.com/cloudflare/workerd/pull/1863 does introduce a material change – the per-Workers request limit has been removed entirely since we determined that the other limits are sufficient and there is no need to retain it as a configurable limit. The remaining Cache API limits continue to apply so this just removes one row in the table.

**Review by irvinebroque — APPROVED** [review]
@fhanau — I follow now, makes sense

File: content/workers/platform/limits.md (hunk 1 above)
Comment (irvinebroque) [reply content/workers/platform/limits.md:105]: Can something here for cache link over to https://developers.cloudflare.com/cache/concepts/default-cache-behavior/#customization-options-and-limits

File: content/workers/platform/limits.md [hunk 2]
```diff
@@ -92,18 +92,15 @@ On the Unbound billing model, scheduled Workers ([Cron Triggers](/workers/config
...
 {{<table-wrap>}}
 
-| Feature                       | Workers Free  | [Bundled](/workers/platform/pricing/#example-pricing-bundled-usage-model) | [Unbound](/workers/platform/pricing/#example-pricing-unbound-usage-model) | [Standard](/workers/platform/pricing/#example-pricing-standard-usage-model)  |
-| ----------------------------- | ------------- | ------- | ------- | ------- |
-| [Max object size](#cache-api-limits) | 512 MB | 512 MB  | 512 MB  | 512 MB  |
-| [Calls/request](#cache-api-limits)   | 50     | 50      | 1,000   | 1,000   |
-| [Storage/request](#cache-api-limits) | 5 GB   | 5 GB    | 5 GB    | 5 GB    |
+| Feature                       | Workers Free  | [Bundled](/workers/platform/pricing/#example-pricing-bundled-usage-model) | [Unbound](/workers/platform/pricing/#example-pricing-unbound-usage-model) and [Standard](/workers/platform/pricing/#example-pricing-standard-usage-model) |
+| ----------------------------- | ------------- | ------- | ------- |
+| [Max object size](#cache-api-limits) | 512 MB | 512 MB  | 512 MB  |
```
Comment (deadlypants1973) [inline content/workers/platform/limits.md]:
```suggestion
| [Maximum object size](#cache-api-limits) | 512 MB | 512 MB  | 512 MB  |
```

**Review by deadlypants1973 — APPROVED** [review]

**Pull request merged**